### DISCUSSION TITLE
Save thumbnails of cloud only assets locally (to improve offline use case)

### DISCUSSION THREAD

**Discussion by radh21301** [immich-app/immich#5720] [Feature Request]
Right now, all pictures and videos that are in the cloud are fetched as we scroll through the timeline. This handicaps the offline functionality since the only things we can see are the assets locally on the device. All cloud only assets show an error icon. 

It would be really nice and useful if we can have the low quality thumbnails saved on the device. It would most certainly make offline use more pleasant!

**Comment by bo0tzz** [comment]
#4466
